### PR TITLE
 FIX #70 Ability to get all refined values : helper.getRefinements()

### LINKED ISSUE #70: Ability to get all refined values
Something like `helper.getRefinements(facet) -> ["value1", "value2", ...]`

### PR DESCRIPTION
**Pull request by bobylito**

### CODE REVIEW

File: src/SearchParameters/index.js [hunk 1]
```diff
@@ -312,6 +312,30 @@ SearchParameters.prototype = {
     } );
   },
   /**
+   * Get the list of conjunctive refinements for a single facet
+   * @param {string} facetName
+   * @return {string[]} list of refinements
+   */
+  getConjunctiveRefinements : function( facetName ) {
+    return this.facetsRefinements[ facetName ];
```
Comment (redox) [inline src/SearchParameters/index.js]: I would append `|| []` to ease its usage. What do you think? (same for all others)

File: src/algoliasearch.helper.js [hunk 2]
```diff
@@ -343,6 +343,61 @@ AlgoliaSearchHelper.prototype.getCurrentPage = function() {
...
+        operator : operator,
+        type : "numeric"
+      } );
+    } );
+  }
+
+  return refinements;
+};
+
 ///////////// PRIVATE
```
Comment (redox) [inline src/algoliasearch.helper.js]: Oh OK, so here it returns an array even if it doesn't exist. Could make sense to only have it here!

File: src/algoliasearch.helper.js [hunk 3]
```diff
@@ -343,6 +343,61 @@ AlgoliaSearchHelper.prototype.getCurrentPage = function() {
   return this.state.page;
 };
 
+/**
+ * Get the list of refinements for a give facet.
```
Comment (redox) [inline src/algoliasearch.helper.js]: `s/facet/attribute/` I would say, because of the numericFilters and the `_tags` very soon.

**Comment by bobylito** [comment]
Could make sense overall. Making the changes!

File: src/algoliasearch.helper.js (hunk 2 above)
Comment (bobylito) [inline src/algoliasearch.helper.js]: Where do you put your comment?! I can't outdate it :)

**Comment by bobylito** [comment]
Seems like we're all done here. Merging!

**Pull request merged**